tests/integration/tables/crontab.cpp

```diff
@@ -50,7 +50,7 @@ TEST_F(Crontab, test_sanity) {
                             {"hour", CronValuesCheck(0, 23)},
                             {"day_of_month", CronValuesCheck(1, 31)},
                             {"month", CronValuesCheck(1, 31, month_list)},
-                            {"day_of_week", CronValuesCheck(0, 6, days_list)},
+                            {"day_of_week", CronValuesCheck(1, 7, days_list)},
```

Comment:
Is this change intentionally in this PR?

Reply:
Yes this is intentional, since test case failed for me. I believe range should be 0-7 not 0-6 or 1-7.
So made change.
              field          allowed values
              -----          --------------
              minute         0-59
              hour           0-23
              day of month   1-31
              month          1-12 (or names, see below)
              day of week    0-7 (0 or 7 is Sun, or use names)